xcell/mappers/mapper_DESY1wl.py

```diff
@@ -133,7 +175,7 @@ def get_signal_map(self, mode=None):
         self.signal_map = self.maps[mod]
         return self.signal_map
 
-    def get_nz(self):
+    def get_nz(self, dz=0):
```

Comment:
You have to add the lines that do something with the `dz`. Copy those in `mapper_dummy.py`